### ISSUE TITLE
TypeError: Class constructor MessageBuilder cannot be invoked without 'new'

### ISSUE THREAD

**Issue by Aus10Caron** [jos-b/webhook-discord#21]
Using your standard example of:
```
const msg = webhook.MessageBuilder()
                .setName("Username")
                .setColor("#aabbcc")
                .setText("This is my webhook!")
                .addField("This", "is")
                .addField("my", "webhook!")
                .setImage("Image url")
                .setTime();

Hook.send(msg);
```

Gives the title TypeError

**Comment by Aus10Caron** [comment]
Just needed to put 'new' in front of webhook.MessageBuilder()

**Comment by jos-b** [comment]
Ahh, not a bug but I do need to fix that in the README. Thanks for letting me know!

**Comment by jos-b** [comment]
Fixed in 6c75ce3d8015de8fce20537230abbaf4bf2c67a4. Released in 2d9bdebcf0ad889a46bad37373d365d84ce51e5e.
